tests/UI/campaigns/functional/FO/06_homePage/02_displaySomeProducts.ts

```diff
@@ -0,0 +1,192 @@
+
+      const popularProductTitle = await homePage.getBlockTitle(page, 2);
+      await expect(popularProductTitle).to.equal('On sale');
+    });
+
+    it('should check the number of products in sale', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkNumberOfProductsInSale', baseContext);
+
+      const productsNumber = await homePage.getProductsBlockNumber(page, 2);
+      await expect(productsNumber).to.equal(2);
+    });
+
+    it('should check All products for sale link', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkAllProductsInSaleLink', baseContext);
+
+      await homePage.goToAllProductsBlockPage(page, 2);
+
+      const pageTitle = await pricesDropPage.getPageTitle(page);
+      await expect(pageTitle).to.equal(pricesDropPage.pageTitle);
+    });
+
+    it('should go to home page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToHomePage2', baseContext);
+
+      await homePage.goToHomePage(page);
+
+      const isHomePage = await homePage.isHomePage(page);
+      await expect(isHomePage, 'Home page is not displayed').to.be.true;
+    });
+  });
+
+  describe('Check new products block', async () => {
+    it('should check new products title', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkNewProductsBlock', baseContext);
+
+      const popularProductTitle = await homePage.getBlockTitle(page, 3);
+      await expect(popularProductTitle).to.equal('New products');
+    });
+
+    it('should check the number of new products', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkNewProductsNumber', baseContext);
+
+      const productsNumber = await homePage.getProductsBlockNumber(page, 3);
+      await expect(productsNumber).to.equal(8);
+    });
+
+    it('should check All new products', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkAllNewProductsLink', baseContext);
+
+      await homePage.goToAllProductsBlockPage(page, 3);
+
+      const pageTitle = await newProductsPage.getPageTitle(page);
+      await expect(pageTitle).to.equal(newProductsPage.pageTitle);
+    });
+
+    it('should go to home page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToHomePage3', baseContext);
+
+      await homePage.goToHomePage(page);
+
+      const isHomePage = await homePage.isHomePage(page);
+      await expect(isHomePage, 'Home page is not displayed').to.be.true;
+    });
+  });
+
+  describe('Check best sellers block', async () => {
+    it('should check best sellers block title', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkBestSellerProducts', baseContext);
+
+      const popularProductTitle = await homePage.getBlockTitle(page, 4);
+      await expect(popularProductTitle).to.equal('Best Sellers');
```

Comment:
Can't use `BestSales.pageTitle` ?

Reply:
It's not a page title, it's a block title in home page :)